app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -273,8 +273,13 @@ class RegisterController @Inject() (val messagesApi:             MessagesApi,
     )
   }
 
-  private def submissionFailureToString(failure: SubmissionFailure): String =
+  private def submissionFailureToString(failure: SubmissionFailure): String = {
     s"Account creation failed. ErrorId: ${failure.errorMessageId.getOrElse("-")}, errorMessage: ${failure.errorMessage}, errorDetails: ${failure.errorDetail}"
+  }
+
+  def showPrivacyPage: Action[AnyContent] = unprotected { implicit request ⇒ implicit htsContext ⇒
```

Comment:
This kind of feels like it belongs more in the `IntroductionController` rather than in the `RegisterController` - though it probably doesn't really belong there either, but it'd be a better place for now